orchagent/orch.cpp

```diff
@@ -13,6 +13,8 @@
 #include "zmqconsumerstatetable.h"
 #include "sai_serialize.h"
 
+#define CONSUMER_POP_MAX_BATCH_COUNT 8
```

Comment:
i do not understand this POP MAX_BATCH, it looks like another batch. We should only have one batch size. hard to reasoning if we have another batch control.

Reply:
I will revert this change to the original fix

Reply:
Change reverted